src/ImageProcessor/ImageFactory.cs

```diff
@@ -75,7 +75,7 @@ public class ImageFactory : IDisposable
         /// Whether to preserve exif metadata. Defaults to false.
         /// </param>
         public ImageFactory(bool preserveExifData = false)
-            : this(preserveExifData, true)
+            : this(preserveExifData, false)
```

Comment:
Why is the `fixGamma` parameter changed here?

Reply:
Because `LinearResize` is buggy an this triggers that over `CompositeResize`. An unfortunate change but unavoidable.